doozer/doozerlib/backend/rebaser.py

```diff
@@ -216,6 +216,14 @@ def _resolve_parents(self, metadata: ImageMetadata, dfp: DockerfileParser, image
         for builder in image_from.get("builder", []):
             parents.append(builder)
         parents.append(image_from)
+
+        # Remove duplicates
+        temp = []
+        for parent in parents:
+            if parent not in temp:
+                temp.append(parent)
+        parents = temp
```

Comment:
@vfreex Looks like one duplicate was getting into the list, so removed it, but if there is a better way we can refactor to that

Reply:
hmm actually this failed rebase for a couple of images (job [run](https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com/job/aos-cd-builds/job/build%252Focp4-konflux/42/console))

Reply:
@ashwindasr I checked the build log and looks like builders are added to the `parents` array twice. Removing line 216-217 should solve this.

Reply:
Thanks!